playbooks/swim_workflow_manager.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: Configure device credentials on Cisco DNA Center
+  hosts: localhost
+  connection: local
+  gather_facts: no
+  vars_files:
+    - "input_swim.yml"
+    - "credentials.yml"
+  tasks:
+    - name: Import an image, tag it as golden and load it on device
+      cisco.dnac.swim_workflow_manager:
+        dnac_host: "{{dnac_host}}"
+        dnac_username: "{{dnac_username}}"
+        dnac_password: "{{dnac_password}}"
+        dnac_verify: "{{dnac_verify}}"
+        dnac_port: "{{dnac_port}}"
+        dnac_version: "{{dnac_version}}"
+        dnac_debug: "{{dnac_debug}}"
+        dnac_log: true
+        dnac_log_level: DEBUG
+        config_verify: true
+        config:
+        - import_image_details:
+            # type: "local"
```

Comment:
Can you please remove all the commented items?

Reply:
removed.